apps/condo/domains/ticket/schema/TicketReportService.js

```diff
@@ -0,0 +1,82 @@
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const { Ticket, TicketStatus } = require('@condo/domains/ticket/utils/serverSchema')
+const moment = require('moment')
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+const access = require('@condo/domains/ticket/access/Ticket')
+
+const PERIOD_TYPES = ['week', 'month', 'quarter']
+
+const countTicketsByStatuses = async (context, statues, dateStart, dateEnd) => {
+    const answer = {}
+    for (const status of statues) {
+        const queryByStatus = [{ createdAt_lte: dateEnd }, { createdAt_gte: dateStart }, { status: { name: status } }]
```

Comment:
You need to use organization in queryByStatus or you will get a count for all tickets